modules/apps/commerce/commerce-account-service/src/main/java/com/liferay/commerce/account/internal/service/CommerceChannelAccountEntryRelAccountEntryLocalServiceWrapper.java

```diff
@@ -160,16 +161,17 @@ public List<AccountEntry> getUserAccountEntries(
 		return new ArrayList<>(new LinkedHashSet<>(userAccountEntries));
 	}
 
-	@Reference(unbind = "-")
-	public void serviceSetter(
-		AccountEntryLocalService accountEntryLocalService) {
-
-		setWrappedService(accountEntryLocalService);
+	@Activate
+	protected void activate() {
+		setWrappedService(_accountEntryLocalService);
```

Comment:
@JanisZhang Please double check if this is needed. Since it is a `ServiceWrapper`, our framework should be able to automatically set the actual service.